### PR TITLE
Adjust chip list component to have multiple values at once

### LINKED ISSUE #6684: Allow entering comma-separated list of IP CIDRs in the network configuration allowlists
**What is the problem, where is it and why is it an issue?**
Currently, it is only possible to add IP CIDRs one by one during Cluster creation, adding a comma-separated list results in a validation error.
image
Having a big amount of CIDRs this can be a tedious task.

**Suggest how to fix it, if possible**

It would be a good UX improvement, to allow adding a comma-separated list of IPs and append them in the allowlist fields.

### PR DESCRIPTION
**Pull request by ahmadhamzh**
**What this PR does / why we need it**:
Adjust the km-chip-list component so that the user can add multiple values separated by commas or spaces all at once.

**Which issue(s) this PR fixes**:
Fixes #6684

**What type of PR is this?**
/kind cleanup
/kind design

```release-note
NONE
```

```documentation
NONE
```

### CODE REVIEW

File: modules/web/src/app/shared/components/chip-list/component.ts [hunk 1]
```diff
@@ -112,7 +112,14 @@ export class ChipListComponent implements OnChanges, OnDestroy, ControlValueAcce
     }
 
     event.chipInput?.clear();
-    this.tags.push(value);
+    const delimiters = [',', ' '];
+    const splitRegex = new RegExp(`[${delimiters.join('')}]`);
+    if (splitRegex.test(value)) {
+      const multiTags = value.split(splitRegex);
+      multiTags.forEach(val => this.tags.push(val.trim()));
```
Comment (Waseem826) [inline modules/web/src/app/shared/components/chip-list/component.ts:119]: If both `,` and ` ` appear together in the string then it `split` will give an extra empty value . For example, `123, 456` would return `['123', '', '456']`. So we should add a filter on splitted values:
```suggestion
      const multiTags = value.split(splitRegex);
      multiTags.filter(val => !!val.trim()).forEach(val => this.tags.push(val.trim()));
```

**Review by ahmedwaleedmalik — APPROVED** [review]
Other than waseem's comment, looks good 👍 
/hold 
to resolve the feedback

**Pull request merged**